.github/workflows/benchmark-core.yml

```diff
@@ -0,0 +1,71 @@
+---
+name: Benchmark - core
+
+on:
+  pull_request:
+    branches: [master]
+    paths:
+      - .github/workflows/benchmark-core.yml
+      - "packages/yew/**"
+      - "tools/benchmark-core/**"
+
+jobs:
+  benchmark-core:
+    name: Benchmark - core
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout master
+        uses: actions/checkout@v3
+        with:
+          repository: "yewstack/yew"
+          ref: master
+          path: yew-master
+
+      - name: Checkout pull request
+        uses: actions/checkout@v3
+        with:
+          path: current-pr
+
+      - name: Setup toolchain
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: stable
+
+      - name: Restore Rust cache for master
+        uses: Swatinem/rust-cache@v2
+        with:
+          working-directory: yew-master
+          key: master
+
+      - name: Restore Rust cache for current pull request
+        uses: Swatinem/rust-cache@v2
+        with:
+          working-directory: current-pr
+          key: pr
+
+      - name: Run pull request benchmark
+        run: cargo bench -q > ../output.log
+        working-directory: current-pr/tools/benchmark-core
+
+      - name: Run master benchmark
+        run: cargo bench -q > ../output.log
+        continue-on-error: true
+        working-directory: yew-master/tools/benchmark-core
```

Comment:
We can probably cache this somewhere (perhaps in firestore db or storage bucket, since we already use firebase) but this is fine for now

Reply:
isnt it already cached by the 2 previous steps (swatinem/rust-cache)?

Reply:
That's true. I was thinking of just downloading the expected output file so benchmark is never run when it can be avoided

Reply:
oh... well. tbh I dont think it's worth the effort haha